src/portable_python/__init__.py

```diff
@@ -652,6 +656,15 @@ def compile(self):
                     else:
                         os.environ[k] = v
 
+    def _apply_patches(self):
+        if patches := self.cfg_patches():
+            for patch in patches:
+                if runez.DRYRUN:
+                    print(f"Would apply patch: {patch}")
```

Comment:
Minor: looks like `patch_file()` was naturally never called before in dryrun mode, but this could go in the `patch_file()` function itself (so that other places don't have to do/repeat `if dryrun`)